CHANGELOG.md

```diff
@@ -10,7 +10,7 @@
 
 ### Adds
 
-* Adds a `last` option to fields. Setting `last: true` on a field puts that field at the end of the field's group order (unless more than one field has that option active).
+* Adds a `last` option to fields. Setting `last: true` on a field puts that field at the end of the field's widget order. If more than one field has that option active the true last item will depend on general field registration order. If the field is ordered with the `fields.order` array, that specified order will take precedence.
```

Comment:
The docs don't speak to what happens if the project-level developer configures the field in an explicit group at project level (should be the same outcome, an explicit order mentioning the field beats "last").

Reply:
```
fields: {
  Groups: {
     seo: {
       // if 'two' was last: true it doesn't matter because the developer now has a preference
       fields: [ 'one', 'two', 'three' ]
    }
  }
}  
```